### PR TITLE
Many to many

### PR DESCRIPTION
**Pull request by yosefHesham**
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@ insert into animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
 ('Devimon','May 12, 2017',11,true,5)
 insert into animals (name,date_of_birth,weight_kg,neutered,escape_attempts)
  values 
-('Charmander','Feb 2, 2020',-11,false,0),
+('Charmander','Feb 2, 2020',-11,false,0) ,
```
Comment (Felix45) [inline data.sql:10]: - [ Required ] Please consider adding `INSERT` statements in the `data.sql` file for the `vets` table as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
Screenshot from 2023-01-21 02-31-57

- [ Required ] Please consider adding `INSERT` statements in the `data.sql` file for the `specializations` table as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
Screenshot from 2023-01-21 02-35-35

- [ Required ] Please consider adding `INSERT` statements in the `data.sql` file for the `visits` table as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
Screenshot from 2023-01-21 02-37-26

File: queries.sql [hunk 2]
```diff
@@ -33,7 +33,7 @@ COMMIT;
 select * from animals;
 select count(*) from animals;
 
-select count(*) from animals where escape_attempts = 0;
+select count(*) from animals where escape_attempts = 0 ;
```
Comment (Felix45) [inline queries.sql:68]: - [ Required ] Please consider adding `SQL QUERY` statements in the `queries.sql` file that answer the following questions as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
Screenshot from 2023-01-21 02-42-52

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @yosefHesham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Highlights :heavy_check_mark: 
- [ ] Professional readme file :heavy_check_mark: 
- [ ] Descriptive pull request title and summary :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @yosefHesham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You well to make the changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**